modules/combined/test/tests/exception/nonad.i

```diff
@@ -0,0 +1,153 @@
+[Problem]
+  register_objects_from = 'TensorMechanicsTestApp'
+  library_path = '../../../../../../moose/modules/tensor_mechanics/test/lib'
+[]
+
+[GlobalParams]
+  order = SECOND
+  displacements = 'disp_x disp_y'
+[]
+
+[Problem]
+  coord_type = RZ
+[]
+
+[Mesh]
+  patch_update_strategy = iteration
+  [./gen]
+    type = FileMeshGenerator
+    file = mesh_in.e
+  [../]
+[]
+
+[Variables]
+  [./disp_x]
+  [../]
+  [./disp_y]
+  [../]
+  [./temp]
```

Comment:
I'd be surprised if you really need the thermal stuff in here to trigger this failure. It would be good to avoid putting tests in the combined module if they don't really need to be there.

Reply:
At least with how this is setup now, it does in fact need `ThermalContact` otherwise the simulation converges and no exceptions are ever thrown

Reply:
The thermal things are necessary, but there is probably other ways to get the same problems, I just couldn't find one. This input file is trimmed down as much as possible to reproduce the problem.

Side note, I used the action unwrapper, but there was a separate error, so there seems to be something screwy with the thermal contact action, but that is a separate problem.

Reply:
Replacing the thermal contact blocks from the output from DumpObjectsProblem using:

```
../../../combined-opt -i nonad.i Problem/type=DumpObjectsProblem Problem/dump_path=ThermalContact/thermal_contact --allow-test-objects
```

results in 
```
Error! Can only use Cholesky decomposition with symmetric positive definite matrices
```

It doesn't have anything to do with this PR, but probably should be dug out what is missing...